administration/src/bp-modules/cards/hooks/useCardGenerator.ts

```diff
@@ -174,11 +174,11 @@ const useCardGenerator = (region: Region): UseCardGeneratorReturn => {
     async (applicationIdToMarkAsProcessed?: number) => {
       await generateCards(
         (codes: CreateCardsResult[], cards: Card[]) => generatePdf(codes, cards, projectConfig.pdf, region),
-        'berechtigungskarten.pdf',
+        cards.length === 1 ? `${cards[0].fullName}.pdf` : 'berechtigungskarten.pdf',
```

Comment:
:question: Should we replace spaces with underscores or hyphens?

Reply:
I think spaces are not critical (from a security perspective -- maybe from a convenience perspective). But maybe (from a security perspective) we should remove non ASCII characters (and potentially convert umlauts etc. to their "base" character) for the file name.
Something like 
```js
  fullName.replace(/ /g, "_").normalize('NFKD').replace(/[^a-z,A-Z,_]/g, '')
```